lib/vector-core/src/config/global_options.rs

```diff
@@ -55,6 +56,16 @@ pub struct GlobalOptions {
     )]
     pub log_schema: LogSchema,
 
+    /// Telemetry options.
+    ///
+    /// Determines whether `source` and `service` tags should be emitted with the
+    /// `component_sent_*` and `component_received_*` events.
+    #[serde(
+        default,
+        skip_serializing_if = "crate::serde::skip_serializing_if_default"
+    )]
+    pub telemetry: Telemetry,
```

Comment:
I'd expected to see cue changes corresponding to this new configuration. Am I missing something?

Reply:
That's a good catch.